netket/sampler/metropolis.py

```diff
@@ -237,6 +237,9 @@ def __pre_init__(self, hilbert, rule, **kwargs):
             machine_pow: The power to which the machine should be exponentiated to generate the pdf (default = 2).
             dtype: The dtype of the statees sampled (default = np.float32).
         """
+        if "n_chains" not in kwargs and "n_chains_per_rank" not in kwargs:
+            kwargs["n_chains"] = 16
```

Comment:
I'd rather set the default to be `n_chains_per_rank`

Reply:
Ok, let's also change this default behavior